xmldoc/filters/options/transform-timeless.xml

```diff
@@ -0,0 +1,4 @@
+<para>
+This option tells
+GPSBabel to creating points without creation times instead of copying the creation time from the source point.
```

Comment:
* to create, 

I like this filter. Useful for KML/Earth, too,  since the Time handling there is confusing.

LGTM